### PR TITLE
hard task

### PR DESCRIPTION
**Pull request by ArturZhukovets**
God bless whoever can read this

### CODE REVIEW

File: Tasks/Zhukovets/Task2/Task2.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        user_choice = input("Какая опция вас интересует?\n"
+                            "Если вы хотите узнать текущее время - нажмите '1'\n"
+                            "Если вы хотите задать время - нажмите '2'\n"
+                            ":")
+        if user_choice == '1':
+            time_string = datetime.datetime.now().strftime("%H:%M")
+            return time_convertor(time_string)
+        elif user_choice == '2':
+            time_string = input("Пожалуйста введите интересующее вас время в формате 'ЧЧ:ММ'\n:")
+            if time_string.split(":")[0].isdigit() and time_string.split(":")[-1].isdigit():
```
Comment (ArturZhukovets) [inline Tasks/Zhukovets/Task2/Task2.py:102]: validation added

File: Tasks/Zhukovets/Task2/Task2.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    18: ['восемнадцать', 'восемнадцати', 'восемнадцать'],
+    19: ['девятнадцать', 'девятнадцати', 'девятнадцать'],
+    20: ['двадцать', 'двадцати', 'двадцать'],
+    }
+tens = {
+    20: 'двадцать',
+    30: 'тридцать',
+    40: 'сорок',
+    50: 'пятьдесят',
+    }
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Zhukovets/Task2/Task2.py:30]: it's a very good data structure

File: Tasks/Zhukovets/Task2/Task2.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                            "Если вы хотите узнать текущее время - нажмите '1'\n"
+                            "Если вы хотите задать время - нажмите '2'\n"
+                            ":")
+        if user_choice == '1':
+            time_string = datetime.datetime.now().strftime("%H:%M")
+            return time_convertor(time_string)
+        elif user_choice == '2':
+            try:
+                time_string = input("Пожалуйста введите интересующее вас время в формате 'ЧЧ:ММ'\n: ")
+                validation_value1, validation_value2 = time_string.split(":")[0], time_string.split(":")[-1]
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Zhukovets/Task2/Task2.py:103]: and what if there is no ":" symbol in the string? you should've check it beforehand, -1 point for the unwise choice

**Review by MikitaTsiarentsyeu — APPROVED** [review]
The solution is correct, and the functional approach looks very good here, I'm gladly adding 9 points to your bank. and the score is 100

**Pull request merged**